src/BigBlueButton.php

```diff
@@ -116,6 +116,14 @@ class BigBlueButton
      */
     public function __construct(?string $baseUrl = null, ?string $secret = null, ?TransportInterface $transport = null, string $hashingAlgorithm = HashingAlgorithm::SHA_1)
     {
+        if (empty($baseUrl)) {
+            @trigger_error(sprintf('Constructing "%s" without passing a server base URL is deprecated and will throw an exception 6.0.', self::class), \E_USER_DEPRECATED);
+        }
+
+        if (empty($secret)) {
+            @trigger_error(sprintf('Constructing "%s" without passing a secret is deprecated and will throw an exception 6.0.', self::class), \E_USER_DEPRECATED);
+        }
+
         // Keeping backward compatibility with older deployed versions
```

Comment:
I think, we should trigger a notice when this is present in the environment to ensure that developers remove their hacks to get around this :) .

Reply:
So just replacing `\E_USER_DEPRECATED` with `\E_USER_NOTICE` ?

Reply:
No, *deprecated* is fine and will be displayed by common reporting tools like Symfony profiler. *notice* would cause throwing an exception e.g. with Symfony error handler.

Reply:
Ok. So what should I change? Do you want to make the change?

Reply:
I would just compare if `getenv` returns something else than `false` for both variables and would trigger a deprecation notice then that reminds the consumer to stop passing environment variables for security reasons.